ms2query/create_sqlite_database.py

```diff
@@ -0,0 +1,407 @@
+        that are the identifier of inchikeys. Default = 'identifier_1'
+    col_name_identifier2:
+        Name of the second column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_2'
+    col_name_score:
+        Name of the third column of the table, this column will store the
+        tanimoto scores. Default = 'tanimoto_score'
+    """
+    create_table_command = f""";
+    DROP TABLE IF EXISTS {table_name};
+    CREATE TABLE {table_name} (
+        {col_name_identifier1} INTEGER,
+        {col_name_identifier2} INTEGER,
+        {col_name_score} REAL,
+        PRIMARY KEY ({col_name_identifier1}, {col_name_identifier2})
+        );
+    """
+    conn = sqlite3.connect(sqlite_file_name)
+    cur = conn.cursor()
+    cur.executescript(create_table_command)
+    conn.commit()
+    conn.close()
+
+
+def add_tanimoto_scores_to_sqlite_table(sqlite_file_name: str,
+                                        npy_file_path: str,
+                                        col_name_identifier1: str =
+                                        'identifier_1',
+                                        col_name_identifier2: str =
+                                        'identifier_2',
+                                        col_name_score: str = 'tanimoto_score'
+                                        ):
+    """Adds tanimoto scores from npy file to sqlite table
+
+    The tanimoto scores are added to a sqlite file with in the first and
+    second column the identifiers that correspond to the index of the rows and
+    columns in the .npy file. The third row contains the tanimoto score. All
+    duplicates are removed, before adding to the sqlite table.
+
+    Args:
+    -------
+    sqlite_file_name:
+        Name of sqlite file to which the table should be added.
+    npy_file_path:
+        Path to the numpy file containing the tanimoto scores. The numpy file
+        is expected to contain a matrix filled with floats with nr of columns =
+        nr of rows.
+    col_name_identifier1:
+        Name of the first column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_1'
+    col_name_identifier2:
+        Name of the second column of the table, this column will store numbers
+        that are the identifier of inchikeys. Default = 'identifier_2'
+    col_name_score:
+        Name of the third column of the table, this column will store the
+        tanimoto scores. Default = 'tanimoto_score'
+    """
+    tanimoto_score_matrix = np.load(
+        npy_file_path,
+        mmap_mode='r')
+
+    # Create a list of consecutive numbers, later used as identifiers for the
+    # inchikeys
+    list_of_numbers = []
+    for i in range(len(tanimoto_score_matrix[0])):
+        list_of_numbers.append(i)
+
+    start_time = time.time()
+    # Create a dataframe and add to sqlite table row by row
```

Comment:
```suggestion
    # Create a dataframe and add to sqlite table row by row
```

Reply:
I would remove the time part in this function. If a user was interested in getting the execution time the function could simple be but in-between such commands.

Reply:
Nice solution! I implemented it and set the default disable_progress_bar to False, so it will show the bar.